src/DependencyInjection/Configuration.php

```diff
@@ -37,9 +38,9 @@ public function __construct(array $lockingStorageAliases)
      */
     public function getConfigTreeBuilder()
     {
-        $treeBuilder = new TreeBuilder();
+        $treeBuilder = new TreeBuilder('task');
 
-        $treeBuilder->root('task')
+        $this->getRootNodeWithoutDeprecation($treeBuilder, 'task')
```

Comment:
Would not create this strange static public function. Would go the same way as in all bundles with:

```
        $treeBuilder = new TreeBuilder('task');

        if (method_exists($treeBuilder, 'getRootNode')) {
            $rootNode = $treeBuilder->getRootNode();
        } else {
            // BC layer for symfony/config 4.1 and older
            $rootNode = $treeBuilder->root('task');
        }
```